docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -516,21 +516,26 @@ As the core components use emotion as a styled engine, the props used by emotion
   +<Collapse classes={{ root: 'collapse' }}>
   ```
 
-### CssBaseline
+### CssBaseline
 
-- The component was migrated to use the `@material-ui/styled-engine` (`emotion` or `styled-components`) instead of `jss`. You should remove the `@global` key when defining the style overrides for it.
+- The component was migrated to use the `@material-ui/styled-engine` (`emotion` or `styled-components`) instead of `jss`. You should remove the `@global` key when defining the style overrides for it and use the CSS template syntax.
```

Comment:
We could revert this change, but I think it's better if we lave it here..

Reply:
Thoughts:

- "and use the CSS template syntax." is not a breaking change, developers aren't required to change it.
- What should be encouraged?
- Should the migration guide only cover required breaking changes or be extended to the encouraged new approaches? I think that if we extend it to new encouraged approaches, it should be very clear that it's not breaking.

I guess this case is special as there is already a @global breaking change.

```suggestion
- The component was migrated to use the `@material-ui/styled-engine` (`emotion` or `styled-components`) instead of `jss`. You should remove the `@global` key when defining the style overrides for it. You could also start using the CSS template syntax over the JavaScript object syntax.
```

Reply:
Agree, let me rephrase 👍